proto/opencdc/v1/opencdc.proto

```diff
@@ -0,0 +1,83 @@
+syntax = "proto3";
+
+package opencdc.v1;
+
+import "google/protobuf/descriptor.proto";
+import "google/protobuf/struct.proto";
+
+// CreatedAt can contain the time when the record was created in the 3rd party
+// system. The expected format is a unix timestamp in nanoseconds.
+option (metadata_created_at) = "opencdc.createdAt";
+// ReadAt can contain the time when the record was read from the 3rd party
+// system. The expected format is a unix timestamp in nanoseconds.
+option (metadata_read_at) = "opencdc.readAt";
+
+// We are (ab)using custom file options to define constants.
+// See https://github.com/protocolbuffers/protobuf/issues/3520#issuecomment-323613839
+extend google.protobuf.FileOptions {
+  string metadata_created_at = 1000;
+  string metadata_read_at = 1001;
+}
+
+// Operation defines what triggered the creation of a record.
+enum Operation {
+  OPERATION_UNSPECIFIED = 0;
+  // Records with operation create contain data of a newly created entity.
+  OPERATION_CREATE = 1;
```

Comment:
Create does sound better than insert, so maybe we should update the design doc too.